app/uk/gov/hmrc/uknwauthcheckerapi/errors/ApiErrorResponse.scala

```diff
@@ -71,16 +71,22 @@ case object InternalServerApiError extends ApiErrorResponse {
   val message:    String = ApiErrorMessages.internalServerError
 }
 
+case object NotAcceptableApiError extends ApiErrorResponse {
+  val statusCode: Int    = NOT_ACCEPTABLE
+  val code:       String = ApiErrorCodes.notAcceptable
+  val message:    String = ApiErrorMessages.notAcceptable
+}
+
 case object NotFoundApiError extends ApiErrorResponse {
   val statusCode: Int    = NOT_FOUND
   val code:       String = ApiErrorCodes.matchingResourceNotFound
   val message:    String = ApiErrorMessages.matchingResourceNotFound
 }
 
-case object NotAcceptableApiError extends ApiErrorResponse {
-  val statusCode: Int    = NOT_ACCEPTABLE
-  val code:       String = ApiErrorCodes.notAcceptable
-  val message:    String = ApiErrorMessages.notAcceptable
+case object RequestEntityTooLargeError extends ApiErrorResponse {
```

Comment:
Please can you name this to follow the other errors RequestEntityTooLargeApiError